lib/ui/tiles/society_tile_left.dart

```diff
@@ -57,12 +57,14 @@ class SocietyTileLeft extends StatelessWidget {
               ),
               Expanded(
                 child: Padding(
-                  padding: const EdgeInsets.all(4.0),
+                  padding: const EdgeInsets.all(0.0),
```

Comment:
Padding `0.0` makes no sense, either remove this or add padding.

Reply:
Fixed

Reply:
Fixed